src/App.tsx

```diff
@@ -1,42 +1,42 @@
-import React, { useMemo } from 'react';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
-
-import { BrukerDetaljerPage } from './component/page/bruker-detaljer/BrukerDetaljerPage';
-import { TiltakinstansDetaljerPage } from './component/page/tiltakinstans-detaljer/TiltakinstansDetaljerPage';
-import StoreProvider from './store/store-provider';
-import { LoginPage } from './component/page/login/LoginPage';
-import { Spinner } from './component/felles/spinner/Spinner';
-import { fetchInnloggetAnsatt } from './api/tiltak-api';
-import { TiltakinstansOversiktPage } from './component/page/tiltakinstans-oversikt/TiltakinstansOversiktPage';
-import { Banner } from './component/felles/menu/Banner';
-import { isNotStartedOrPending, isRejected, usePromise } from './utils/use-promise';
-import { AxiosResponse } from 'axios';
-import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
-import { hentSisteLagretEllerForsteTilgjengeligVirksomhet } from './store/valgt-virksomhet-store';
-import { InnloggetAnsatt, Virksomhet } from './domeneobjekter/ansatt';
+import { AxiosResponse } from 'axios'
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper'
+import React, { useMemo } from 'react'
```

Comment:
Nå ble react importen flytta ned, det er ikke helt bra

Reply:
Det er egentlig ingen grunn til at React skal ligge på toppen annet enn at vi er vant til det

Reply:
Det fungerer uansett, grunnen til at den burde ligge øverst er at det da er veldig tydelig at det er en react komponent

Reply:
Nja. Det er ikke det man burde se på hvis man lurer på om det er en komponent

Reply:
Dette går under samme kategori som det å fjerne ";". Det er litt uvant, men det blir bedre på sikt når man er vant til det

Reply:
Hvorfor er det bedre på sikt å ha react importen i midten?

Reply:
Imports er sortert alfabetisk. Det er ikke noe spesielt med React importen, så den burde ikke trenge å få spesialhåndtering

Reply:
Ok. Er det en fordel å sortere de alfabetisk? Rekkefølgen på imports handler i fleste tilfeller om hva man foretrekker og hva som blir lesbart.

Reply:
Det er ikke mulig å få til med mindre man lager en custom plugin. Og det er ikke mulig (eller verdt) å prøve å vedlikeholde det

Reply:
Jeg tenker at automatisk sortering gir mer verdi enn å ikke ha det, selv om man kan argumentere for at sorteringen ikke er perfekt. Det er umulig å vedlikeholde konvensjoner for importering hvis ikke det gjøres automatisk

Reply:
Jeg vet at man kan få til å få react imports på toppen hvertfall men er ikke sikker på det andre. Uansett så er hovedpoenget her at dette er en endring som automatisk gjør at rekkefølgen på imports blir(etter min mening) dårligere/mindre lesbar.